samples/quickperf/src/main/java/com/google/cloud/jdbc/quickperf/config/Config.java

```diff
@@ -0,0 +1,132 @@
+package com.google.cloud.jdbc.quickperf.config;
```

Comment:
nit: here and elsewhere, add a copyright header to all files

Reply:
added copyright notes